cmd/otel-agent/main.go

```diff
@@ -3,105 +3,22 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build otlp
-
 // otel-agent is a binary meant for testing the comp/otelcol package to ensure that it is reusable
```

Comment:
Do you need to update this comment?

Reply:
we need to, but will do after RFC is approved.